packages/typespec-go/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@
 * Fixed some cases where a client name could stutter.
 * Force the body paramter to be required for `PATCH` and `PUT` operations.
 
+### Features Added
+
+* When Go tools are found on the path, the following steps happen after successfully generating code.
+  * Execute `gofmt -w .`.
```

Comment:
Could we do `goimports -w .` instead? I know I have some packages that generate with broken inputs, so I always run go imports in my build.go file instead of go fmt

Reply:
`goimports` isn't part of the standard Go toolset, so there's no guarantee it will be installed. While we _could_ `go install` it, I'd prefer to avoid that (this is already a slippery slope IMO, and having the emitter make network calls increases the grade).

Can you please let me know what imports are broken? If this is a codegen bug we should fix it.

Reply:
Let me emphasize that we _must_ fix such bugs. Emitted SDKs should compile as-is without the need to execute `goimports`.

Reply:
I generally use goimports because sometimes changes you make will make it so you no longer need some imports. 

I think it's okay if we say that it needs to be installed prior to the emitter running, similar to how we require Go be installed.

Reply:
> I generally use goimports because sometimes changes you make will make it so you no longer need some imports.
> 
> I think it's okay if we say that it needs to be installed prior to the emitter running, similar to how we require Go be installed.

Ditto on what Richard said. Some of the transforms can effect imports. Would be nice to have this tool run as part of the process.

Reply:
Sounds good to me!

Reply:
Fixed.